package.json

```diff
@@ -132,22 +131,21 @@
     "deprecated-react-native-prop-types": "^2.3.0",
```

Comment:
```suggestion
```
After the removal of `native-base` and the upgrade to `react-native-pdf` which was still using `ViewPropTypes` on the current version, this dependency is not used anymore. I can see you already removed the patch as well. Does it makes sense to you?

Reply:
We can make a try, anyway the dependency should be anyway installed as a dependency in any of the updated dependency, so I guess we can safely remove it from our dependency

Reply:
addressed in [6239cff](https://github.com/pagopa/io-app/pull/5864/commits/6239cffc2c5f00804fb2fe85daa1fb2e6b374127)

Reply:
Great, thanks.